reconstruction/rtpc/src/main/java/org/jlab/service/rtpc/RTPCEngine.java

```diff
@@ -102,13 +101,15 @@ public boolean processDataEvent(DataEvent event) {
             return true;
         }
 
-         int runNo = 10;
+        int runNo = 10;
+        int eventNo = 777;
```

Comment:
same as above

Reply:
they must be initialized here. Removing the definition and the initialization would crashing building coatjava. 

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project clas12detector-rtpc: Compilation failure: Compilation failure: 
[ERROR] /w/hallb-scshelf2102/clas12/mhattawy/clara-bonusdev/clas12-offline-software/reconstruction/rtpc/src/main/java/org/jlab/service/rtpc/RTPCEngine.java:[121,77] variable runNo might not have been initialized
[ERROR] /w/hallb-scshelf2102/clas12/mhattawy/clara-bonusdev/clas12-offline-software/reconstruction/rtpc/src/main/java/org/jlab/service/rtpc/RTPCEngine.java:[136,77] variable eventNo might not have been initialized

Reply:
Taken care of in commit https://github.com/JeffersonLab/clas12-offline-software/pull/833/commits/729a5f5afbe86c4c8943fafd63304f48d8d7e89c